packages/hs-react-ui/src/components/Button/Button.test.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import styled from 'styled-components';
+import { action } from '@storybook/addon-actions';
+import { render, fireEvent, waitFor, screen } from '@testing-library/react';
```

Comment:
I think styled and action aren't necessary here - not sure about all the resting-library imports though.